Template Variable updates (#5274)
DataDog/documentation: content/en/graphing/dashboards/template_variables.md

* A **Name** *-Mandatory-*:

>     Value of your Template Variable displayed in your graphs query.

Reviewer: ```suggestion
    The value of your template variable displayed in your graph query.
```

* A **Tag or Attribute** *-Mandatory-*:

    * For Tag: If your tag follow [the tags best practice][1] with a `key:value` format, the *Tag* is the `key` of your tags.

>     * For Attribute: Use a [facet or a Measure as Template variables](#template-variables-with-logs-and-apm-queries).

Reviewer: ```suggestion
    * Attribute: Use a [facet or measure as the template variable](#template-variables-with-logs-and-apm-queries).
```

> * A **Default Value** *-Optional-*:

Reviewer: ```suggestion
* **Default Value** *-optional-*:
```

>     Default value for your Template Variable Tag or Attribute.

Reviewer: ```suggestion
    The default value for your template variable tag or attribute.
```

Once created, notice that you have statistics upon your template variables usage in your graphs. In the picture below, the template variable is not used in both graph of the dashboard:

{{< img src="graphing/dashboards/template_variables/stats_tv.png" alt="statistic TV" responsive="true" style="width:70%;">}}

Decide if you want to remove/add this template variable to all of your graph widgets with the respective **Remove From All** and **Add to All** buttons.

> #### Template variables with Logs and APM queries

Reviewer: ```suggestion
#### Template variables with logs and APM queries
```

> Template variables work on log and APM query based widgets—since metrics, logs, and APM share the same tags.

Reviewer: ```suggestion
Template variables work with log and APM query based widgets because metrics, logs, and APM share the same tags.
```

> Additionally, it is possible to define specific log/APM template variables based on your [log][2] or APM facets. These template variables start with `@`.

Reviewer: ```suggestion
Additionally, you can define log or APM template variables based on [log][2] or APM facets. These template variables start with `@`.
```

{{< img src="graphing/dashboards/template_variables/log_template_variables.png" alt="log template variables" responsive="true" style="width:85%;">}}

> **Note**: Using the `Add to all` button adds this template variable to all log/APM widgets only.

Reviewer: ```suggestion
**Note**: Using the `Add to all` button adds the template variable to all log/APM widgets.
```
